src/include/platform/DeviceInstanceInfoProvider.h

```diff
@@ -142,6 +142,18 @@ class DeviceInstanceInfoProvider
      *          if access fails.
      */
     virtual CHIP_ERROR GetRotatingDeviceIdUniqueId(MutableByteSpan & uniqueIdSpan) = 0;
+
```

Comment:
So far there's no common code that would use this value, and I think the idea was to let applications initialize handlers of TestEventTriggers, so we may simply implement it in our class and not add it to the common interface.